model/feature_flags.go

```diff
@@ -7,8 +7,12 @@ type FeatureFlags struct {
 	// Exists only for unit and manual testing.
 	// When set to a value, will be returned by the ping endpoint.
 	TestFeature string
+
+	// Toggle on and off scheduled jobs for cloud user limit emails
+	CloudEmailJobsEnabledMM29999 bool
```

Comment:
In the past when I've done feature flags I found it useful to include the ticket number that caused the flag to be created so that people can go back to Jira and get a bigger picture of what that feature flag might control

What are your thoughts on that?

Reply:
I personally feel is a bit verbose to have on the actual variable name and I haven't seen this pattern in the server code base. I do think the ticket number is valuable so I would be more inclined to put it in the actual variable definition as a comment.